docs/faq.md

```diff
@@ -18,9 +27,29 @@ assertions. The ability to search for elements (using search actions) makes test
 UI changes. For example, EarlGrey provides APIs that allow searching for elements in scrollable
 containers, regardless of the amount of scrolling required.
 
+**Why do the tests have the application scaled with borders around it? How can I get them to fit in
+the video frame?**
+
+For your tests to have the application properly scaled, make sure the app under test has correct
+launch screen images present for all supported devices (see 
+[iOS Developer Library, Launch Files](https://developer.apple.com/library/ios/documentation/UserExperience/Conceptual/MobileHIG/LaunchImages.html)).
+
+**Why does EarlGrey need to modify the test's scheme and add a Copy Files Build Phase?**
```

Comment:
Can we add this to `ifaq.md` for infrequently asked questions? This came up as part of the AWS eval. It'd be nice to have an official answer instead of sending people to SO or Google. Other iOS frameworks don't require this setup so I think it makes sense to explain why this is necessary.